packages/tools/generator-component/generators/app/prompts.js

```diff
@@ -55,6 +55,20 @@ const prompts = [
             return answers.componentType === 'uiComponent';
         }
     },
+    {
+        message: 'Does your component require Bundlewatch checks?',
+        name: 'needsBundlewatch',
+        type: 'confirm',
+        default: true,
+    },
+    {
+        message: "What size limit do you want in place for your bundles (in kB)? e.g. '15'",
```

Comment:
Is it possible to have a default value for this based on the type of component? Or set a very strict 5kB for new packages which can be increased later?

Reply:
Will set 5kB as the default for now :)